components/automate-ui/src/app/app.module.ts

```diff
@@ -209,6 +209,8 @@ import { WelcomeModalComponent } from './page-components/welcome-modal/welcome-m
 import { WarningBannerComponent } from './page-components/warning-banner/warning-banner.component';
 import { AppConfigService } from 'app/services/app-config/app-config.service';
 import { DataFeedCreateComponent } from './pages/data-feed-create/data-feed-create.component';
+import { DataFeedTableComponent
+} from './page-components/data-feed-table/data-feed-table.component';
```

Comment:
```javascript
import { 
   DataFeedTableComponent
} from './page-components/data-feed-table/data-feed-table.component';
```

You can write it like this as this is like the universal syntax we are following